File: AvroToJson/ComponentTest/Models/OutputEventJson.cs
using Newtonsoft.Json;

namespace AvroToJson.ComponentTest.Models;

public class OutputEventJson
{
    [JsonProperty("deviceId")]
    public string DeviceId { get; set; }

    [JsonProperty("value")]
    public double Value { get; set; }

    [JsonProperty("timestamp")]
    public long Timestamp { get; set; }
}

File: AvroToJson/src/main/java/cheetah/example/model/json/OutputEventJson.java
package cheetah.example.model.json;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class OutputEventJson {
    private String deviceId;
    private double value;
    private long timestamp;
}
